docs/sample_config.yaml

```diff
@@ -784,22 +784,33 @@ caches:
   #
   #cache_entry_ttl: 30m
 
-  # This flag enables cache autotuning, and is further specified by the sub-options `max_cache_memory_usage`,
-  # `target_cache_memory_usage`, `min_cache_ttl`. These flags work in conjunction with each other to maintain
-  # a balance between cache memory usage and cache entry availability. You must be using jemalloc to utilize
-  # this option, and all three of the options must be specified for this feature to work.
+  # This flag enables cache autotuning, and is further specified by the
```

Comment:
to me, "flag" is a boolean. None of these things are flags?  Not a major problem though.

Reply:
s/flag/setting/ would be better, I think